client/src/components/Header/index.js

```diff
@@ -9,12 +9,13 @@ import {
 } from "@material-ui/core";
 
 const StyledText = styled.h1`
-  background-image: linear-gradient(white, black);
+  background-image: linear-gradient(#2196f3, #21cbf3);
```

Comment:
Really like the use of linear-gradient, makes it look really cool. Not a serious issue but does it need to be background-image? I think just background would work

Reply:
Changed it to just background and it still works!